utils/src/main/scala/za/co/absa/enceladus/utils/fs/FileSystemVersionUtils.scala

```diff
@@ -164,6 +164,15 @@ class FileSystemVersionUtils(conf: Configuration) {
     fs.listStatus(hdfsPath).map(_.getPath.toString)
   }
 
+  /**
+    * Deletes a directory and all its contents recursively
+    */
+  def deleteDirectoryRecursively(path: String): Unit = {
+    log.warn(s"Deleting '$path' recursively...")
```

Comment:
Isn't the warning little late here?
If it's potentially problematic, perhaps it should be announced upon start the user to have soemthing do about it? Or it's just a normal step of the processing... 🤔

Reply:
This is more for further log analytics and monitoring in my opinion. Remember the idea is that these are automated, nobody's gonna watch potentially thousands of jobs run every day :)

Reply:
In that case, if it's a part of "normal" processing, I would think INFO severity might be more suitable. But not a hard request, just an opinion :-)

Reply:
It also looks to me that INFO level is more appropriate here

Reply:
I agree, `INFO` seems more appropriate.